bash-setup.sh

```diff
@@ -29,3 +29,5 @@ curl -s "https://get.sdkman.io" | bash
 source "$HOME/.sdkman/bin/sdkman-init.sh"
 sdk install gradle
 sdk install groovy
+
+cp -r ./zsh-functions $HOME/.oh-my-zsh/lib
```

Comment:
we should all decide on a standard way we have zsh setup.

Reply:
That's the default way it gets set up iirc

Reply:
I don't think I have that file. I do have .oh-my-zsh, and then all my custom stuff is in `.omz-custom`

Reply:
@johntosberg rounding back on this, it seems like `.oh-my-zsh` is the default folder for stuff, but ZSH does allow you to specify a folder for your custom stuff. 

e.g. my zshrc has

```
ZSH_CUSTOM=~/.omz-custom
```

Is that something we want to do here to contain all the 'custom' stuff on a machine?